package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.11.2",
   "description": "JavaScript tooling for working with WebAssembly Components",
   "author": "Guy Bedford",
+  "engines" : { 
+    "node" : ">=18"
```

Comment:
I generally prefer to leave this out, because it's just such a bad bad UX for users. Rather users can just hit the bug when running on 16. It's EOL next month anyway so I have no problem with that.

So please let's keep the package.json unrestricted here.